### ISSUE TITLE
index.php

### ISSUE THREAD

**Issue by crislinux** [crislinux/Prueba#1]
<!DOCTYPE html>
<html>
<head>
<script src="jquery-3.3.1.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   	
<script type="text/javascript" charset="utf-8">	
	var $ =jQuery.noConflict();
	  $(document).ready(function() {
	  // Parametros para el combo
	   $("#regionlocation").change(function () {
	      $("#regionlocation option:selected").each(function () {
	        obselect=$(this).val();
	        $.post("combo50.php",{obselect:obselect},function(data){
	        $("#regionlocation").html(data);  
	      });     
	     });
	   });
	     });
	
</script> 

    
</head>
<body>
<center>
<h1>Formulario de Votacion</h1>
<br>
<section>
  <form method="POST" id="form_ajax" action="">
  <label for = "Nombre y Apellido">
    <span>Nombre y Apellido </span>
    <input type="text" name="nameandlastname" id="nameandlastname" placeholder="Nombre y Apellido" />
    <div id="errorname"></div>
  </label>
  <br><br>
  <label for = "Alias">
    <span> Alias </span>
    <input type="text" name="alias" id="alias" placeholder="Alias" />
    <div id="erroralias"></div>
  </label>
<br><br>
  <label for = "Rut">
    <span>Rut </span>
    <input type="text" name="dni" id="dni" placeholder="DNI" />
    <div id="errordni"></div>
  </label>
<br><br>
  <label for = "Email">
    <span>Email </span>
    <input type="text" name="email" id="email" placeholder="Email" />
    <div id="erroremail"></div>
  </label>
<br><br>
  <label for = "Region">
    <span>Region </span>
    <select name="regionlocation" id="regionlocation"></select>
  </label>
<br><br>
  <label for = "Comuna">
    <span>Comuna </span>
    <select name="communelocation" id="communelocation"></select>
  </label>
  <br><br>
  <label for = "Candidato">
    <span>Candidato </span>
    <select name="candidate" id="candidate">
    <option value="1">Candidato A</option>
    <option value="2">Candidato B</option>
    </p></select>
  </label>
  <br><br>
    <P>Como se entero de Nosotros</P>
    <input type="checkbox" name="web" value="web"> WEB <td>
    <input type="checkbox" name="tv" value="tv"> TV <td>
    <input type="checkbox" name="socialnetwork" value="socialnetwork"> REDES SOCIALES<td>
    <input type="checkbox" name="friend" value="friend"> AMIGO<br>
    <input type="hidden" name="ajax">
    <p><input type="submit" id="save" value="Votar" /></p>
  </form>
</section>
<br><br>
<div id="response"></div>
</center>
</body>
<script>
$(function()
$('#save').click(function()
{
    var nameandlastname = document.getElementById('nameandlastname').value;
    var alias= document.getElementById('alias').value;
    var dni= document.getElementById('dni').value;
    var email= document.getElementById('email').value;
    var regionlocation= document.getElementById('regionlocation').value;
    var communelocation= document.getElementById('communelocation').value;
    var candidate= document.getElementById('candidate').value;

    var ruta= "name="+nameandlastname+"&alias="+alias+"&dni"+dni+"&email"+email+"&region"+regionlocation+"&commune"+communelocation+"&candidate"+candidate;
    $.ajax({
      type: "POST",
      url: 'server.php',
      data: {ruta},
    })
    .done (function(res){
      $('#response').html)(res)

    })
    .fail (function(){
      console.log("error");

    })
    .always(function(){
      console.log("complete");

    })
});
    </script>  

</html>

**Comment by crislinux** [comment]
<!DOCTYPE html>
<html>
<head>
<script src="jquery-3.3.1.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   	
<script type="text/javascript" charset="utf-8">	
	var $ =jQuery.noConflict();
	  $(document).ready(function() {
	  // Parametros para el combo
	   $("#regionlocation").change(function () {
	      $("#regionlocation option:selected").each(function () {
	        obselect=$(this).val();
	        $.post("combo50.php",{obselect:obselect},function(data){
	        $("#regionlocation").html(data);  
	      });     
	     });
	   });
	     });
	
</script> 



    
</head>
<body>
<center>
<h1>Formulario de Votacion</h1>
<br>
<section>
  <form method="POST" id="form_ajax" action="">
  <label for = "Nombre y Apellido">
    <span>Nombre y Apellido </span>
    <input type="text" name="nameandlastname" id="nameandlastname" placeholder="Nombre y Apellido" />
    <div id="errorname"></div>
  </label>
  <br><br>
  <label for = "Alias">
    <span> Alias </span>
    <input type="text" name="alias" id="alias" placeholder="Alias" />
    <div id="erroralias"></div>
  </label>
<br><br>
  <label for = "Rut">
    <span>Rut </span>
    <input type="text" name="dni" id="dni" placeholder="DNI" />
    <div id="errordni"></div>
  </label>
<br><br>
  <label for = "Email">
    <span>Email </span>
    <input type="text" name="email" id="email" placeholder="Email" />
    <div id="erroremail"></div>
  </label>
<br><br>
  <label for = "Region">
    <span>Region </span>
    <select name="regionlocation" id="regionlocation"></select>
  </label>
<br><br>
  <label for = "Comuna">
    <span>Comuna </span>
    <select name="communelocation" id="communelocation"></select>
  </label>
  <br><br>
  <label for = "Candidato">
    <span>Candidato </span>
    <select name="candidate" id="candidate">
    <option value="1">Candidato A</option>
    <option value="2">Candidato B</option>
    </p></select>
  </label>
  <br><br>
    <P>Como se entero de Nosotros</P>
    <input type="checkbox" name="web" value="web"> WEB <td>
    <input type="checkbox" name="tv" value="tv"> TV <td>
    <input type="checkbox" name="socialnetwork" value="socialnetwork"> REDES SOCIALES<td>
    <input type="checkbox" name="friend" value="friend"> AMIGO<br>
    <input type="hidden" name="ajax">
    <p><input type="submit" id="save" value="Votar" /></p>
  </form>
</section>
<br><br>
<div id="response"></div>
</center>
</body>
<script>
$(function()
$('#save').click(function()
{
    var nameandlastname = document.getElementById('nameandlastname').value;
    var alias= document.getElementById('alias').value;
    var dni= document.getElementById('dni').value;
    var email= document.getElementById('email').value;
    var regionlocation= document.getElementById('regionlocation').value;
    var communelocation= document.getElementById('communelocation').value;
    var candidate= document.getElementById('candidate').value;

    var ruta= "name="+nameandlastname+"&alias="+alias+"&dni"+dni+"&email"+email+"&region"+regionlocation+"&commune"+communelocation+"&candidate"+candidate;
    $.ajax({
      type: "POST",
      url: 'server.php',
      data: {ruta},
    })
    .done (function(res){
      $('#response').html)(res)

    })
    .fail (function(){
      console.log("error");

    })
    .always(function(){
      console.log("complete");

    })
});
    </script>  


</html>

**Comment by crislinux** [comment]
<!DOCTYPE html>
<html>
<head>
<script src="jquery-3.3.1.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   	
<script type="text/javascript" charset="utf-8">	
	var $ =jQuery.noConflict();
	  $(document).ready(function() {
	  // Parametros para el combo
	   $("#regionlocation").change(function () {
	      $("#regionlocation option:selected").each(function () {
	        obselect=$(this).val();
	        $.post("combo50.php",{obselect:obselect},function(data){
	        $("#regionlocation").html(data);  
	      });     
	     });
	   });
	     });
	
</script> 



    
</head>
<body>
<center>
<h1>Formulario de Votacion</h1>
<br>
<section>
  <form method="POST" id="form_ajax" action="">
  <label for = "Nombre y Apellido">
    <span>Nombre y Apellido </span>
    <input type="text" name="nameandlastname" id="nameandlastname" placeholder="Nombre y Apellido" />
    <div id="errorname"></div>
  </label>
  <br><br>
  <label for = "Alias">
    <span> Alias </span>
    <input type="text" name="alias" id="alias" placeholder="Alias" />
    <div id="erroralias"></div>
  </label>
<br><br>
  <label for = "Rut">
    <span>Rut </span>
    <input type="text" name="dni" id="dni" placeholder="DNI" />
    <div id="errordni"></div>
  </label>
<br><br>
  <label for = "Email">
    <span>Email </span>
    <input type="text" name="email" id="email" placeholder="Email" />
    <div id="erroremail"></div>
  </label>
<br><br>
  <label for = "Region">
    <span>Region </span>
    <select name="regionlocation" id="regionlocation"></select>
  </label>
<br><br>
  <label for = "Comuna">
    <span>Comuna </span>
    <select name="communelocation" id="communelocation"></select>
  </label>
  <br><br>
  <label for = "Candidato">
    <span>Candidato </span>
    <select name="candidate" id="candidate">
    <option value="1">Candidato A</option>
    <option value="2">Candidato B</option>
    </p></select>
  </label>
  <br><br>
    <P>Como se entero de Nosotros</P>
    <input type="checkbox" name="web" value="web"> WEB <td>
    <input type="checkbox" name="tv" value="tv"> TV <td>
    <input type="checkbox" name="socialnetwork" value="socialnetwork"> REDES SOCIALES<td>
    <input type="checkbox" name="friend" value="friend"> AMIGO<br>
    <input type="hidden" name="ajax">
    <p><input type="submit" id="save" value="Votar" /></p>
  </form>
</section>
<br><br>
<div id="response"></div>
</center>
</body>
<script>
$(function()
$('#save').click(function()
{
    var nameandlastname = document.getElementById('nameandlastname').value;
    var alias= document.getElementById('alias').value;
    var dni= document.getElementById('dni').value;
    var email= document.getElementById('email').value;
    var regionlocation= document.getElementById('regionlocation').value;
    var communelocation= document.getElementById('communelocation').value;
    var candidate= document.getElementById('candidate').value;

    var ruta= "name="+nameandlastname+"&alias="+alias+"&dni"+dni+"&email"+email+"&region"+regionlocation+"&commune"+communelocation+"&candidate"+candidate;
    $.ajax({
      type: "POST",
      url: 'server.php',
      data: {ruta},
    })
    .done (function(res){
      $('#response').html)(res)

    })
    .fail (function(){
      console.log("error");

    })
    .always(function(){
      console.log("complete");

    })
});
    </script>  


</html>
